data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table.py

```diff
@@ -0,0 +1,130 @@
+"""
+Original Issues: DC-1012
+
+Background
+In order to avoid further changes to the standard OMOP person table, three non-standard fields will be housed in a
+person_ext table.
+
+Cleaning rule script to run AFTER deid.
+This cleaning rule will populate the person_ext table
+The following fields will need to be copied from the observation table:
+src_id (from observation_ext, should all be “PPI/PM”)
+state_of_residence_concept_id: the value_source_concept_id field in the OBSERVATION table row where
+observation_source_concept_id  = 1585249 (StreetAddress_PIIState)
+state_of_residence_source_value: the concept_name from the concept table for the state_of_residence_concept_id
+person_id (as research_id) can be pulled from the person table
+"""
+
+# Project imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.bq_utils import WRITE_TRUNCATE
+from utils.bq import JINJA_ENV
+
+# Query to create person_ext table
+PERSON_EXT_TABLE_QUERY = JINJA_ENV.from_string("""
+SELECT p.person_id, e.src_id,
+o.value_source_concept_id AS state_of_residence_concept_id,
+c.concept_id AS state_of_residence_source_concept_id,
```

Comment:
This wasn't asked for in the ticket.

Reply:
I have no idea where I thought I saw that... Removing now!